src/app/src/pages/redteam/setup/components/Plugins.tsx

```diff
@@ -20,28 +20,29 @@ import TextField from '@mui/material/TextField';
 import Typography from '@mui/material/Typography';
 import { alpha } from '@mui/material/styles';
 import {
+  AGENTIC_EXEMPT_PLUGINS,
```

Comment:
this is a great change but I wish it happened outside of this PR, no need to change now. We should update the linter rules.